packages/docs/content/6_patterns/step-list.mdx

```diff
@@ -0,0 +1,126 @@
+---
+title: Step List
+---
+
+import { Badge } from '@cmsgov/design-system';
+
+A step list represents a user's progression through an application or multi-page
+form. It serves as a table of contents and a way to quickly see where they are
+and what they should be working on next.
+
+## Start, Resume, and Edit links
+
+A user interacts with the steps through "Start", "Resume", and "Edit"
+links.
+
+In the React component, the links' `href` properties are determined by the
+`step` object's `href` property. One can also optionally pass an
+`onStepLinkClick` function that will cancel the default link behavior and call
+`onStepLinkClick` with `href` as a parameter for apps that handle routing with
+JavaScript.
+
+When a step has substeps and is incomplete, the `href` property should be set to
+match the `href` of the current substep—that is, the first incomplete substep.
+The "Edit" button will only appear on substeps that have been completed.
+
+## `<StepList>`
+
+The `StepList` component is the preferred method for building a step list, since it handles all the state logic necessary to produce its markup. A step is represented by an object with text, progress, and routing information and can optionally include an array of substeps as well as a description.
+
+<StorybookExample
+  componentName="step-list"
+  sourceFilePath="components/StepList/StepList.tsx"
+  storyId="patterns-step-list--step-list-example"
+/>
+
+### Props
+
+<PropTable componentName="StepList" />
+
+## `<Step>`
+
+### Step Object
```

Comment:
This is a weird situation because the `StepObject` is an interface that we export that is used to configure StepList data. However, it is not the same thing as the props for the `Step` component, so we cannot use react-docgen to grab this data programmatically.

In the current doc site, this data is hard coded into the content, so I did the same here until an alternative solution can be found